commit 8e86c883fdd629b3e423e3635e116c3465ad8ad0 (refs/changes/99/1499/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-10T16:42:17-08:00 (2 years, 8 months ago)
    
    joyent/node-triton#170 Integration tests fail to change company name back to original value when it contains spaces

diff --git a/test/integration/cli-account.test.js b/test/integration/cli-account.test.js
index 04b42b1..ff0cf33 100644
--- a/test/integration/cli-account.test.js
+++ b/test/integration/cli-account.test.js
@@ -13,6 +13,7 @@
  */
 
 var h = require('./helpers');
+var format = require('util').format;
 var test = require('tape');
 
 
@@ -100,11 +101,13 @@ test('triton account', function (tt) {
 
     tt.test(' triton account update companyName=<oldvalue>', writeTestOpts,
             function (t) {
-        h.triton('account update companyName=' + account.companyName || '',
-                function (err, _o, _e) {
-            if (h.ifErr(t, err))
-                return t.end();
-            t.end();
-        });
+        h.triton(
+            format('account update companyName=\'%s\'',
+                account.companyName || ''),
+            function (err, _o, _e) {
+                if (h.ifErr(t, err))
+                    return t.end();
+                t.end();
+            });
     });
 });
